core/src/main/scala/sttp/tapir/Validator.scala

```diff
@@ -111,11 +112,17 @@ object Validator extends ValidatorMacros {
   case class Pattern[T <: String](value: String) extends Primitive[T] {
     override def doValidate(t: T): ValidationResult = ValidationResult.validWhen(t.matches(value))
   }
-  case class MinLength[T <: String](value: Int) extends Primitive[T] {
-    override def doValidate(t: T): ValidationResult = ValidationResult.validWhen(t.size >= value)
+  case class MinLength[T <: String](value: Int, countCodePoints: Boolean = false) extends Primitive[T] {
```

Comment:
**suggestion:** A scaladoc explaining the purpose `countCodePoints` may be useful here.